app/pb_kits/playbook/plugins/pb_chart.js

```diff
@@ -1,8 +1,12 @@
 import Highcharts from 'highcharts'
```

Comment:
Let's work on getting this separated to keep this clean

Reply:
Got everything requiring Sass imports moved over into Theme file.